src/site/content/en/blog/squoosh-v2/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Announcing Squoosh v2
+subhead: New codecs support, updatd design, and CLI support!
+description: |
+  Major version change to Squoosh.app with new CLI feature.
+scheduled: true
+date: 2020-12-09
```

Comment:
```suggestion
date: 2020-12-09
authors:
  - kosamari
```
We need you listed as author so that when people read "Squoosh is an app that our team built" they know who is talking